stdlib/LinearAlgebra/test/tridiag.jl

```diff
@@ -859,6 +859,28 @@ end
     @test axes(B) === (ax, ax)
 end
 
+@testset "Reverse operation on Tridiagonal" begin
+    n = 5
```

Comment:
You'll want to do `for n = 5:6` here to test both the even and odd size case for `reverse!`.